libraries/GCS_MAVLink/GCS.cpp

```diff
@@ -113,6 +113,30 @@ void GCS::send_named_float(const char *name, float value) const
                                   (const char *)&packet);
 }
 
+#if HAL_HIGH_LATENCY2_ENABLED
+void GCS::enable_high_latency_connections(bool enabled)
+{
+    // Set HL2 message to be 0.2Hz on all active MAVLink channels
+    for (uint8_t i=0; i<num_gcs(); i++) {
+        GCS_MAVLINK &c = *chan(i);
+        if (enabled) {
+            c.set_mavlink_message_id_interval(MAVLINK_MSG_ID_HIGH_LATENCY2, 5000);
```

Comment:
Why aren't we simply inhibiting all sending instead of this?

Sending could be inhibited by adding a snippet to `comm_send_buffer`

Reply:
Now using ``comm_send_buffer`` instead

Reply:
... so why are we still changing the latency here?